### PR TITLE
changelog: fix typo

### PR DESCRIPTION
**Pull request by FFY00**
Signed-off-by: Filipe Laíns <lains@riseup.net>

### CODE REVIEW

**Comment by FFY00** [comment]
See #285 

Well, seems Github is being weird because there were already CI runs for this commit. I am just gonna merge this. We can check if the CI is being triggered properly next time.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ Unreleased
 - Validate that the supplied source directory is valid (`PR #260`_, Fixes `#259`_)
 - Set and test minimum versions of build's runtime dependencies (`PR #267`_, Fixes `#263`_)
 
-.. _PR #267: https://github.com/pypa/build/pull/260
+.. _PR #267: https://github.com/pypa/build/pull/267
 .. _PR #260: https://github.com/pypa/build/pull/260
 .. _#263: https://github.com/pypa/build/issues/259
```
Comment (henryiii) [inline CHANGELOG.rst:14]: Same typo here. Already fixed in #274

Comment (FFY00) [reply CHANGELOG.rst:14]: Whops, this will conflict. Can you rebase?

Comment (henryiii) [reply CHANGELOG.rst:14]: Sure, usually do each push.

**Pull request merged**